src/lib/build-style.js

```diff
@@ -245,13 +250,42 @@ export const buildStyle = (name, absoluteStylePath, layerDir, options = {}) => {
     console.log(`Building style ${chalk.blue(name)}`);
   }
 
+  // Helper functions for unused context
+  const deleteProp = (object, path) => {
+    var last = path.pop();
+    delete path.reduce((o, k) => o[k] || {}, object)[last];
+  };
+
+  const removeEmpty = obj => {
+    return JSON.parse(
+      JSON.stringify(obj, (k, v) =>
+        isPlainObject(v) && !Object.keys(v).length ? undefined : v
+      )
+    );
+  };
```

Comment:
I have a slight preference for moving these out of this function because `buildStyle` is getting large and it might be useful in terms of testing these.

Reply:
Good call here. Also turned out these weren't performing quite as expected 😅  Added some tests!